acceptance/testdata/pr/pr-view.txtar

```diff
@@ -0,0 +1,25 @@
+# Use gh as a credential helper
```

Comment:
I recognise that this is almost identical to `pr-create-basic.txtar`. I'm being quite intentional about keeping them separate. Just because they look the same now doesn't mean they will always look the same.